ipn/ipnlocal/c2n.go

```diff
@@ -305,7 +305,7 @@ func findCmdTailscale() (string, error) {
 	}
 	switch runtime.GOOS {
 	case "linux":
-		if self == "/usr/sbin/tailscaled" {
+		if self == "/usr/sbin/tailscaled" || self == "/usr/bin/tailscaled" {
```

Comment:
Should we also allow for /usr/local/[s]bin?

Reply:
sure!